app/backend/server/routers/dates_router.py

```diff
@@ -0,0 +1,19 @@
+from fastapi import APIRouter, Query, Depends, HTTPException
+from typing import List
+from datetime import date, time
+from utils.datetime_utils import daterange
+
+from db.models import *
+from db.database import transaction
+from db.crud import get_multiple_users_redused_dates_data
+from db import schemas
+
+router = APIRouter()
+
+
+@router.get("/api/v1/dates_status", response_model=schemas.MultipleUsersDatesResponce)
```

Comment:
we agreed to add an app for v1...
v1 will be a sub application like so:
https://fastapi.tiangolo.com/tutorial/sub-applications-proxy/
and everything under it will be routed by name spaces:
https://fastapi.tiangolo.com/tutorial/bigger-applications/

Reply:
done